lua/cfc_time/server/time/sv_time_tracking.lua

```diff
@@ -40,19 +71,23 @@ function ctime:updateTimes()
         if isValid then
             data.duration = sessionTime
 
-            local sessionId = self.sessions[steamId]
-            batch[sessionId] = data
+            local sessionID = self.sessions[steamID]
+            batch[sessionID] = data
+
+            local timeDelta = now - self.lastUpdate
+            local newTotal = self.totalTimes[steamID] + timeDelta
+            self.totalTimes[steamID] = newTotal
```

Comment:
Deliberated a bit over removing the `initialTimes` table, but I realized the only place we used it was to calculate the total times, so I figured... why not just do that?

So we have a table of `totalTimes` used purely for display logic (never affects anything permanent) that gets incremented by the diff each time instead of calculating `initial + session`.